src/CHANGES.txt

```diff
@@ -18,6 +18,8 @@ Bugs Fixed
 - Pinned the ``transaction`` and ``manuel`` dependencies to Python 2.5-
   compatible versions when installing under Python 2.5.
 
+- Cleanup of nested mvcc connections could have failed.
```

Comment:
The convention is to have the changelog entries in imperative mode.  Something like:

```
- Avoid failure due to repeated cleanup of a connection's storage (typically during shutdown).
```